feature/policy_forwarding/encapsulation/otg_tests/inner_ttl_ipv4/README.md

```diff
@@ -89,6 +206,19 @@ paths:
 
   /network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/ipv6/config/destination-address:
   /network-instances/network-instance/policy-forwarding/policies/policy/rules/rule/ipv6/config/hop-limit:
+  # afts state paths set via gRIBI
+  /network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/id:
+  /network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/next-hop-group-id:
+  /network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/next-hop/state/index:
+  /network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/next-hop/state/network-instance:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/state/index:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/state/type:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/state/mpls/pushed-mpls-label-stack:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/state/udp/src-ip:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/state/udp/dst-ip:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/state/udp/dst-udp-port:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/state/udp/ip-ttl:
+  /network-instances/network-instance/afts/next-hops/next-hop/encap-headers/encap-header/state/udp/dscp:
 
 rpcs:
   gnmi:
```

Comment:
add the gribi rpcs like at:

https://github.com/openconfig/featureprofiles/blob/27b6dc48817c6d6fddd08494f54812cedb702ad2/feature/gribi/otg_tests/mpls_in_udp/README.md?plain=1#L234-L238